js/webrtc.js

```diff
@@ -931,6 +931,14 @@ var spreedPeerConnectionTable = [];
 			if (!OCA.SpreedMe.app.videoDisabled) {
 				OCA.SpreedMe.app.enableVideo();
 			}
+
+			var availableStream = $(OCA.SpreedMe.webrtc.webrtc.localStream.getVideoTracks());
+			var $hideVideoButton = $('#hideVideo');
+			if(availableStream.length < 1 || availableStream.length == undefined) {
```

Comment:
You could write this simpler as `if (!availableStream.length) {` (btw, this is an array, so `length` is never undefined).